components/coding/studentPortal/WorkshopsCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "../../ui/Button";
+
+// type WorkshopsCardProps = {
```

Comment:
This is the right idea, take a look at other components and see how I defined the type for props.

You should uncomment this and add more properties